qiita_pet/support_files/config_portal.txt

```diff
@@ -0,0 +1,41 @@
+# ---------- Base information for the website ----------
```

Comment:
Can you change the extension to `.cfg`? That way the format will be recognized both by github and text editors, for example like [this](https://gist.github.com/ElDeveloper/c0ea363168b6f86cf63f).